### ISSUE TITLE
Support Bitbucket 5

### ISSUE THREAD

**Issue by tm1000** [ef-labs/stash-hook-mirror#45]
Add support for Bitbucket 5

**Comment by herrphon** [comment]
Bitbucket 5 is supported. I just checked and no Java API calls were removed that this plugin is using. So although it's not marked as compatible in the marketplace yet, it'll work just fine.

**Comment by tm1000** [comment]
Agree. I noticed that force enabling it works. But it'd sure be nice to have it "supported" :-)

**Comment by adrianluisgonzalez** [comment]
Sorry for radio silence.  I will get to this soon.  Hopefully this Friday.
